### PR TITLE
LPS-190473 [CP 2.0] Created CP Home page category filter layout

### PR DESCRIPTION
**Pull request by samuelleolopes**
[LPS-148378](https://liferay.atlassian.net/browse/LPS-148378)

### CODE REVIEW

**Comment by samuelleolopes** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsNavbar/components/ProjectCategoryDropdown/ProjectCategoryDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import i18n from '~/common/I18n';
+import {Skeleton} from '~/common/components';
+import getKebabCase from '~/common/utils/getKebabCase';
+import {PROJECT_CATEGORY_GROUPS} from '~/routes/customer-portal/utils/constants/projectCategoryGroups';
+
+const ProjectCategoryDropdown = ({loading}) => {
+	const [active, setActive] = useState(false);
+	const [selectedItemIndex, setSelectedItemIndex] = useState(0);
+
+	const items = [
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsNavbar/components/ProjectCategoryDropdown/ProjectCategoryDropdown.jsx:24]:
```suggestion
	const [selectedProjectCategoryIndex, setSelectedProjectCategoryIndex] = useState(0);

	const projectCategoryItems = [
```

Please change to a more specific name, like projectCategoryItems.

**Review by pedroliveirav — CHANGES_REQUESTED** [review]
Need changes 🔨

**Comment by samuelleolopes** [comment]
ci:test:sf

**Review by EugenioAntunes — APPROVED** [review]
Nice job, @samuelleolopes  ⚡

**Review by pedroliveirav — COMMENTED** [review]
Good job! 🚀

**Review by pedroliveirav — APPROVED** [review]
Good job! 🚀

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:relevant

**Comment by samuelleolopes** [comment]
ci:test:relevant

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**